java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -765,21 +793,39 @@ private void initAndResizeOffsetBuffer(int currentIndex) {
       }
     }
 
-    private void initValidBuffer() {
+    private void initAndResizeValidBuffer() {
       if (this.valid == null) {
```

Comment:
I would like to consolidate the init and resize buffer logic to be in one method that takes the buffer to init and the length but maybe we can do that as a follow on as some of the internal of that are specific to what the buffer is.